.circleci/config.yml

```diff
@@ -31,7 +31,7 @@ jobs:
           name: Install
       - run:
           command: |
-            python -m pytest -n 4 --durations=0 --cov allensdk --cov-report xml
```

Comment:
Was the durations flag only there for testing what was taking so long?

Reply:
yes, it was adding a few thousand lines of text after the tests finished.

Reply:
Wow that's a lot...